generator/tests/cdn/webpack-customizer.spec.ts

```diff
@@ -87,7 +89,7 @@ describe('Test webpack customizer', () => {
         expect(baseConfig.entry['cdn-support']).toEqual(path.resolve(__dirname, '../../src/cdn/bootstrap.js'));
         expect(baseConfig.output.filename).toBe('[name].[chunkhash].js');
         expect(baseConfig.optimization.runtimeChunk).toBe('single');
-        expect(baseConfig.optimization.splitChunks.cacheGroups.vendors).toEqual({
+        expect(baseConfig.optimization.splitChunks.cacheGroups.defaultVendors).toEqual({
```

Comment:
Is this a functional change or just a rename to make the code clearer?

Reply:
It's a part of v4->v5 migration https://webpack.js.org/migrate/5/#update-outdated-options
See `optimization.splitChunks.cacheGroups.vendors → optimization.splitChunks.cacheGroups.defaultVendors`